frontend/js/components/battle-details/BattleInfoDetails.js

```diff
@@ -49,25 +50,27 @@ const BattleMatchesInformation = ({ creatorTeam, opponentTeam, winners }) => {
 };
 
 const BattleInfoDetails = ({ battle, user }) => {
-  const { creator, opponent, winner } = battle;
+  const { creator, opponent, winner, creator_team, opponent_team, matches_winners } = battle;
+
+  if (!creator_team || !opponent_team) return null;
 
   return (
     <div className="content">
       {get(user, 'email') === get(creator, 'email') || winner ? (
```

Comment:
You can use `isEqual` from `lodash`. https://lodash.com/docs/4.17.15#isEqual

```suggestion
      {isEqual(get(user, 'email'), get(creator, 'email')) || winner ? (
```